src/components/Plot.js

```diff
@@ -24,6 +26,15 @@ class PlotlyGraph extends React.Component {
   }
 
   render() {
+    if(this.props.data.length === 0) {
```

Comment:
yes, exactly. but with new implementation using `isFetching` attribute it will disappear once we get some error. But need to think what to display in that case.